src/Microsoft.VisualStudio.Editors/package/VBPackage.vb

```diff
@@ -24,10 +24,14 @@ Namespace Microsoft.VisualStudio.Editors
     '*   from here for the moment.
     '*
     '* In the future, we should consider moving to a RegPkg.exe model
-    <
-        Guid("67909B06-91E9-4F3E-AB50-495046BE9A9A"),
-        ProvideOptionPage(GetType(GeneralOptionPage), "Projects and Solutions", ".NET Core", 1400, 1400, True),
-        CLSCompliant(False)
+    <Guid("67909B06-91E9-4F3E-AB50-495046BE9A9A"),
+    ProvideOptionPage(GetType(GeneralOptionPage),
+                     "Projects and Solutions",
+                     "SDK-Style Projects",
+                     0,        ' categoryResourceID: Not used, we don't own parent category
+                     0,        ' pageNameResourceID: Not used, set in Microsoft.VisualStudio.Editors.pkgdef, 
+                     True),    ' supportsAutomation
```

Comment:
Named arguments might be better here, assuming they work on attribute constructors in VB. e.g. `categoryResourceID := 0`.

Reply:
I couldn't figure out VB's named argument syntax, and it hit a compiler bug that I've not yet filed. Do you know what the syntax is? I thought it was `categoryResourceId::=` but compiler does not like that.

Reply:
it is `:=` but you can't use that here.  Because VB is case-insensitive and `ProvideOptionPageAttribute` has public properties that only differ from its constructor argument names by case it is ambiguous.

Reply:
Ah, that's probably what I hit and resulted in an extraordinary large number of errors.